security/hosts_multitenancy.adoc

```diff
@@ -0,0 +1,105 @@
+[[security-hosts]]
+= Container Hosts and Multi-tenancy
+{product-author}
+{product-version]
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:sourcedir: ../
+
+toc::[]
+
+[[security-hosts-the-os-matters]]
+== The OS Matters
+
+Containers make it easier for developers to build and promote the application
+and its dependencies as a unit. Containers also make it easy to deploy multiple
+applications on a single host, using the kernel and the docker runtime to spin
+up each individual container.
+
+To take full advantage of this packaging and deployment technology, an
+operations team needs the right environment for running containers. Operations
+needs an operating system (OS) that can secure containers at the boundaries:
+securing the host kernel from container escapes and securing containers from
+each other.
```

Comment:
I have no idea what the above is saying. Maybe:

"You need an OS that can secure container so that the host kernel is secure and containers are secure from each other".

Either way, wouldn't it be easier to just say something like "OpenShift is availble on RHEL and Atomic Host, which give X benefits"? I feel like this intro para is telling the reader they can install on anything, "but make sure you can do these secure things, otherwise you'll be in trouble", which I feel is misleading and not correct...